cmake/compiler/gcc/target_warnings.cmake

```diff
@@ -14,8 +14,6 @@ macro(toolchain_cc_warning_base)
 
 if(CMAKE_CXX_COMPILER_VERSION VERSION_GREATER "9.1.0")
   zephyr_compile_options(
-    # FIXME: Remove once #16587 is fixed
-    -Wno-address-of-packed-member
```

Comment:
We shouldn't have an empty zephyr_compile_options() invocation. The whole if-statement should be removed.

Reply:
removed